src/lib/parse/CompoundValueNode.cpp

```diff
@@ -691,32 +694,28 @@ std::string CompoundValueNode::toJson(bool toplevel)
     break;
 
   case orion::ValueTypeVector:
-
-    if (childV.size() == 0)
+    for (unsigned int ix = 0; ix < childV.size(); ix++)
     {
-      out = "[]";
-    }
-    else {
-      out = "[" + childV[0]->toJson(false);
-      for (unsigned int ix = 1; ix < childV.size(); ix++)
-      {
-        out += "," + childV[ix]->toJson(false);
-      }
-      out += "]";
+      jh.addRaw(childV[ix]->toJson(false));
     }
+    out = jh.str();
     break;
 
   case orion::ValueTypeObject:
+    // In thic case we cannot use JsonHelper to build the object, as we don't have a
+    // key-value sequence to invoke addXX() method
```

Comment:
An empty JsonHelper would render as "{}", no?

Reply:
Yes, but I cannot use the helper here. The way the helper works it needs both a key and value to invoke addXX():

```
addXX(key, value)
```

However, in this method I don't have a key because the child toJson() method I'm using returns the key along the value.

Maybe it can be solved with a deeper refactor of CompoundValue::toJson() but that's complex so I left out of this PR.

Reply:
My point was that with an empty object, you wouldn't call any addXX, the loop would be empty and you would end up with '{}' ...  Or not?

Reply:
Not sure what you mean... you mean to use JsonObjectHelper in the case of no child? I mean, something like:

```
    if (childV.size() == 0)
    {
      JsonObjectHelper jh;
      out = jh.str();
    }
```
    
That could be use, but I think is better the current approach. However, I don't have a strong opinion on it, if you prefer to be changed to the above.

Reply:
Getting confused ... Sorry about that.
I wan't really interested in JsonHelper here, only the special treatment of empty objects. My bad. 
Using code instead of words. More exact.

Something like this:

```
out = "{";
unsigned int ix;
for (ix = 1; ix < childV.size(); ix++)
{
  out += childV[0].toJson(false);
  if (ix != childV.size() - 1)  out += ",";
}

out += "}";
```
No special treatment for 0 children.

Reply:
Ok. I'll fix.

Reply:
Fixed in 03e44cf